.github/workflows/archlinux.yml

```
@@ -0,0 +1,23 @@
+name: Build on Archlinux
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    container:
+      image: "archlinux"
+      # These options are required to be able to run lldb inside the container
+      options: "--cap-add=SYS_PTRACE --security-opt seccomp=unconfined"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Dependencies
+        run: pacman -Sy --noconfirm llvm clang cmake make lld lldb gcc libc++ openmp
```

`-u, --sysupgrade` is useful here to avoid a partially upgraded state:

```suggestion
        run: pacman -Syu --noconfirm llvm clang cmake make lld lldb gcc libc++ openmp
```